src/i18n/locales/index.ts

```diff
@@ -0,0 +1,25 @@
+import translationEN from './en';
+import translationPSEUDO from './pseudo';
+
+export enum i18nLanguageCode {
+  ENGLISH = 'en',
+  PSEUDO = 'pseudo'
+}
+
+export const defaultResources = {
+  [i18nLanguageCode.ENGLISH]: {
+    name: 'English',
+    ...translationEN
+  }
+};
+
+export const resources = {
+  [i18nLanguageCode.ENGLISH]: {
+    name: 'English',
+    ...translationEN
+  },
+  [i18nLanguageCode.PSEUDO]: {
+    name: 'Pseudo',
+    ...translationPSEUDO
+  }
```

Comment:
Maybe we only enable this in development mode using a `process.env.NODE_ENV` check?